src/synthcity/plugins/core/models/tabular_encoder.py

```diff
@@ -103,18 +104,14 @@ def _fit_continuous(self, data: pd.Series) -> FeatureInfo:
         )
 
     def fit(
-        self, raw_data: pd.Series, discrete_columns: Optional[List] = None
+        self, raw_data: pd.DataFrame, discrete_columns: Optional[List] = None
```

Comment:
`BinEncoder` works on Series. Are you using it in DDPM?

Reply:
But in Line 119 and 254 there are for name in raw_data.columns. Are they not supposed to be Dataframes?

Reply:
Great catch! Thanks for fixing this.